### ISSUE TITLE
Download with aria2 is failing recently

### ISSUE THREAD

**Issue by cderv** [ScoopInstaller/Main#1481]
It was working correctly and suddently now it fails so I share this with you 

This happens when trying to install ` scoop install hugo-extended` but it errors with other too

```powershell
Installing 'hugo-extended' (0.76.2) [64bit]
Starting download with aria2 ...
Invoke-Expression: C:\Users\chris\scoop\apps\scoop\current\lib\install.ps1:286
Line |
 286 |          Invoke-Expression $aria2 | ForEach-Object {
     |          ~~~~~~~~~~~~~~~~~~~~~~~~
     | This script contains malicious content and has been blocked by your antivirus software.

Checking hash of hugo_extended_0.76.2_windows-64bit.zip ... Get-FileHash: C:\Users\chris\scoop\apps\scoop\current\lib\install.ps1:648
Line |
 648 |  …          return (Get-FileHash -Path $file -Algorithm $algname).Hash.T …
     |                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot find path 'C:\Users\chris\scoop\cache\hugo-extended#0.76.2#https_github.com_gohugoio_hugo_releases_download_v0.76.2_hugo_extended_0.76.2_windows-64bit.zip'
     | because it does not exist.

ERROR You cannot call a method on a null-valued expression.
ERROR Hash check failed!
App:         main/hugo-extended
URL:         https://github.com/gohugoio/hugo/releases/download/v0.76.2/hugo_extended_0.76.2_windows-64bit.zip
Expected:    6cb6fc5809fa1ae9506faa75bee95122946dd48ad4bd07cf5856d5bb108fe4e9
Actual:

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Main/issues/new?title=hugo-extended%400.76.2%3a+hash+check+failed
```

If I deactivate `aria2` then this is working well. 
```powershell
scoop config aria2-enabled false
```

Some powershell update or Windows10 update must have triggered this. The message says malware but I don't have any other antivirus than windows one.

Something to fix with aria2 or else ?

**Comment by dudeamax99** [comment]
Have you tried updating Aria2? I think there was an update recently.

**Comment by cderv** [comment]
I am using last version available from Scoop 
```powershell
❯ scoop update aria2
aria2: 1.35.0-1 (latest version)
Latest versions for all apps are installed! For more information try 'scoop status'
```
No release since 1 year it seems 
https://github.com/aria2/aria2/releases

**Comment by wstein** [comment]
@cderv your screen output shows this line:
```
This script contains malicious content and has been blocked by your antivirus software.
```
So maybe your anti virus software is involved? Could you deactivate it and try it again? What AV are you using?

On my PC it is installing fine:
```batch
scoop install hugo-extended
WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
Installing 'hugo-extended' (0.76.3) [64bit]
Starting download with aria2 ...
Download: [#81b118 128KiB/38MiB(0%) CN:5 DL:533KiB ETA:1m13s]
Download: [#81b118 6.6MiB/38MiB(17%) CN:5 DL:5.1MiB ETA:6s]
Download: [#81b118 17MiB/38MiB(46%) CN:5 DL:7.9MiB ETA:2s]
Download: [#81b118 28MiB/38MiB(74%) CN:4 DL:8.7MiB ETA:1s]
Download: [#81b118 37MiB/38MiB(96%) CN:3 DL:8.7MiB]
Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 81b118|OK  |   8.2MiB/s|D:/scoop/cache/hugo-extended#0.76.3#https_github.com_gohugoio_hugo_releases_download_v0.76.3_hugo_extended_0.76.3_windows-64bit.zip
Download: Status Legend:
Download: (OK):download completed.
Checking hash of hugo_extended_0.76.3_windows-64bit.zip ... ok.
Extracting hugo_extended_0.76.3_windows-64bit.zip ... done.
Linking D:\scoop\apps\hugo-extended\current => D:\scoop\apps\hugo-extended\0.76.3
Creating shim for 'hugo'.
'hugo-extended' (0.76.3) was installed successfully!
```

**Comment by cderv** [comment]
> The message says malware but I don't have any other antivirus than windows one.

I don't have any other AV than Windows defender. 

So I believe a Windows update or Windows defender update added a new rule regarding powershell script execution ?

**Comment by cderv** [comment]
Ok I retried and this now works. I guess the AV was involved and something changed. 🤷‍♂️
